README.md

```diff
@@ -6,7 +6,22 @@
 
 ---
 
-The IntelliJ Ballerina plugin provides the Ballerina development capabilities in IntelliJ IDEA. Click the links below for instructions on how to download, install, and use the features of the IntelliJ plugin.
+## Overview
+The Ballerina IntelliJ plugin provides the [Ballerina programming language](https://ballerina.io/) development capabilities within IntelliJ IDEA. While it offers a subset of features compared to the [Ballerina VSCode extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina), it is designed to meet the specific needs of those working within the IntelliJ ecosystem.
```

Comment:
Isn't it worth mentioning what is the subset of high level features we support in IDEA?

Reply:
We've already mentioned all the high-level features under the `Plugin Guide` section. Do you prefer to duplicate some of them in here as well?

Reply:
No need to duplicate in that case. Can we add a link to the text `subset of features` to direct to the `Using the features of the plugin` sub title of the same page?

Reply:
Yes that makes sense. Updated in https://github.com/ballerina-platform/plugin-intellij/pull/77/commits/60392953353daf6164dd6f76c0cb694794721487